### PR TITLE
[INTR-88] Adding IP Allowlist modification by support to audit event docs

### PR DESCRIPTION
**Pull request by ddmclav**
### What does this PR do? What is the motivation?
This PR adds an entry to our documentation for a new audit event involving IP Allowlist modifications by Datadog Support, which are being added as a part of [Customer Transparency](https://datadoghq.atlassian.net/wiki/spaces/SECENG/pages/2679212146/Customer+Transparency+Expose+Critical+Support+Actions+in+Audit+Trail).

The related PR: https://github.com/DataDog/dd-source/pull/131740

### Merge instructions
The deployment PR referenced above has been merged, and this one can be merged once it has been reviewed.

Please merge after reviewing.

### CODE REVIEW

File: content/en/account_management/audit_trail/events.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | [User invite (Support)][12] | A user was invited to the org by Datadog Support. | `@evt.name:"Access Management" @evt.actor.type:SUPPORT_USER @asset.type:user @action:created` |
 | [User's role (Support)][100] | A user was added or deleted from a role in the org by Datadog Support. | `@evt.name:"Access Management" @evt.actor.type:SUPPORT_USER @asset.type:role @action:modified` |
 | [Role modified (Support)][101] | A role was modified by Datadog Support, and what the previous and new permissions are. | `@evt.name:"Access Management" @evt.actor.type:SUPPORT_USER @asset.type:role @action:modified` |
+| [IP Allowlist Modified (Support)][115] | A new IP was added to the org's IP Allowlist by Datadog Support. | `@evt.name:"Access Management" @evt.actor.type:SUPPORT_USER @asset.type:ip_allowlist @action:modified` |
```
Comment (maycmlee) [inline content/en/account_management/audit_trail/events.md:70]:
```suggestion
| [IP allowlist Modified (Support)][115] | A new IP was added to the org's IP allowlist by Datadog Support. | `@evt.name:"Access Management" @evt.actor.type:SUPPORT_USER @asset.type:ip_allowlist @action:modified` |
```

**Review by maycmlee — COMMENTED** [review]
Small edit

File: content/en/account_management/audit_trail/events.md (hunk 1 above)
Comment (ddmclav) [reply content/en/account_management/audit_trail/events.md]: Thanks for pointing that out! Just readjusted @maycmlee

**Review by tannerprynn-datadog — APPROVED** [review]

**Review by maycmlee — APPROVED** [review]
Thanks, LGTM! Feel free to comment `/merge` to merge the PR.

**Review by maycmlee — APPROVED** [review]

**Comment by maycmlee** [comment]
@ddmclav is this ready to merge?

**Comment by ddmclav** [comment]
> @ddmclav is this ready to merge?

Yes! Sorry for the delay

**Pull request merged**